api/server/structs/conversions.go

```diff
@@ -699,6 +700,11 @@ func (m *SyncCommitteeMessage) ToConsensus() (*eth.SyncCommitteeMessage, error)
 	if err != nil {
 		return nil, server.NewDecodeError(err, "Signature")
 	}
+	// Add validation to check if the signature is valid BLS format
+	_, err = bls.SignatureFromBytes(sig)
+	if err != nil {
+		return nil, server.NewDecodeError(errors.New("Invalid Signature"), "Signature")
```

Comment:
```suggestion
		return nil, server.NewDecodeError(err, "Signature")
```
The second parameter is used to add the field name to the final error message, no need to add it for the second time

Reply:
I have fixed